src/stores/OwnBeaconStore.ts

```diff
@@ -65,6 +65,26 @@ type OwnBeaconStoreState = {
     beaconsByRoomId: Map<Room['roomId'], Set<BeaconIdentifier>>;
     liveBeaconIds: BeaconIdentifier[];
 };
+
+const CREATED_BEACONS_KEY = 'mx_live_beacon_created_id';
+const storeLocallyCreateBeaconEventId = (eventId: string): void => {
+    const ids = getLocallyCreatedBeaconEventIds();
+    window.localStorage.setItem(CREATED_BEACONS_KEY, JSON.stringify([...ids, eventId]));
```

Comment:
Looks like we don't prune old values out of this - is there a risk that this will end up getting too big?

Reply:
good point, i've added some pruning